server/sonar-server/src/main/java/org/sonar/server/db/migrations/PlatformDatabaseMigration.java

```diff
@@ -95,12 +95,7 @@ private void startAsynchronousDBMigration() {
     }
 
     running.set(true);
-    executorService.execute(new Runnable() {
-      @Override
-      public void run() {
-        doDatabaseMigration();
-      }
-    });
+    executorService.execute(this::doDatabaseMigration);
   }
 
   private void doDatabaseMigration() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Private method 'doDatabaseMigration' is never used. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AUnusedPrivateMethod)